addons/popochiu/engine/popochiu.gd

```diff
@@ -312,13 +242,13 @@ func queue(instructions: Array, show_gi := true) -> void:
 		if instruction is Callable:
 			await instruction.call()
 		elif instruction is String:
-			await _eval_string(instruction as String)
+			await PopochiuCharactersHelper.exec_string(instruction as String)
 	
 	if show_gi:
```

Comment:
I guess this should be called `show_gui` at this point?

Reply:
I agree the name could be different, but I don't like `show_gui` because it is also used to trigger an `E.wait()` when the string has only dots (e.g. `"..."`), and that's not related with the GUI at all. I'm gonna think another name.

Reply:
Done.